MUI_Config/MUI_Config.config.lua

```diff
@@ -162,6 +162,12 @@ function C_ConfigModule:GetConfigTable()
                         L["Adjust the width of the main container."], "\n\n", L["Default value is "], "750");
                     dbPath      = "profile.bottomui.width";
                 };
+                {
+                    name = "Main Container Strata",
+                    type = "dropdown",
+                    options = tk.Constants.ORDERED_FRAME_STRATAS,
+                    dbPath = tk.Strings:Concat("profile.bottomui.frameStrata")
```

Comment:
You can change `tk.Strings:Concat("profile.bottomui.frameStrata")` to simply `"profile.bottomui.frameStrata"` because there is nothing to concatenate here.